app/npm-shrinkwrap.json.orig

```diff
@@ -0,0 +1,943 @@
+{
```

Comment:
Doh, so this is something that has started to happen to me as I've been resolving conflicts with a merge tool in the terminal and then committing using desktop. The merge process creates a bunch of these files for the merge tool to consume but doesn't stage them, they then get deleted once the commit is done. Unfortunately, with desktop we see these files as just any new file and auto-select them for inclusion in the commit. We should probably figure out a way to ignore these files or be more clever in our inclusion logic while resolving a conflict